pages/jobs/[slug].tsx

```diff
@@ -51,8 +51,8 @@ const JobPosting: NextPage<JobPageProps> = ({
     return (
         <Layout theme="LIGHT" preview={preview} jobsAvailable={jobsAvailable}>
             <Metadata
-                title={content.metadataTitle}
-                description={content.metadataDescription}
+                title={`${job.title} | Torchbox Careers`}
```

Comment:
I've gone for `| Torchbox Careers` as this is more specific, however across the site we've typically used just `| Torchbox` - should I update this to match the rest of the site?

Reply:
I'd update to " - Jobs | Torchbox" personally, I think this is more about consistency for google results, see below.
<img width="813" alt="Screenshot 2022-11-03 at 09 01 43" src="https://user-images.githubusercontent.com/35934884/199681919-ce5506f1-9a40-4cf1-8ef9-77aa1a4ea3b9.png">
<img width="840" alt="Screenshot 2022-11-03 at 09 01 35" src="https://user-images.githubusercontent.com/35934884/199681925-44d83c74-53b3-41d4-8e25-b9e458e228b9.png">

Reply:
Can we update all use of the "<Metadata" to, perhaps by appending this from the Metadata component instead?

Reply:
`Careers - jobs | Torchbox` might sound a little bit funny, so perhaps just the jobs page should be "- jobs | Torchbox".

I'd go for this change for now and bring it up with the SEO team for the whole site, not just the careers pages as currently there is no consitency.

<img width="664" alt="Screenshot 2022-11-03 at 09 11 25" src="https://user-images.githubusercontent.com/35934884/199683654-d6c54444-85e8-433b-b6b9-cd1dadc01f29.png">